decidim-accountability/app/cells/decidim/accountability/project/show.erb

```diff
@@ -1,4 +1,4 @@
-  <h2 class="h2 decorator"><%= title %></h2>
+  <h1 class="h2 decorator" data-heading-tag><%= title %></h1>
```

Comment:
Bad idea with the drawers as also mentioned at #9830.

Reply:
I'll mark this as resolved as we are talking about the drawers #10184 and a possible removal will take place in a different PR.